lib/rubygems/basic_specification.rb

```diff
@@ -35,7 +35,14 @@ def initialize
   end
 
   def self.default_specifications_dir
```

Comment:
Can move this to under the `class << self` scope?

Reply:
Well, looking at 021d303debfba923dc24dbfa99152c8c708e60c6, it appeared this is acceptable. Because otherwise, it should be probably modified everywhere.

Reply:
But if your proposal is the preferred way, then you are right that it could set bad example in the `basic_specification.rb`

Reply:
I see. I will update them with this patch.